spec.html

```diff
@@ -3658,12 +3658,23 @@ <h1>Well-Known Intrinsic Objects</h1>
             </tr>
             <tr>
               <td>
-                %IteratorPrototype%
```

Comment:
i missed this removal; the intention was to keep pre-existing intrinsics in this table. i'll make a PR to re-add `%IteratorPrototype%` in the future.